client/src/featureform/client.py

```diff
@@ -0,0 +1,75 @@
+from typing import Union
+from .register import (
+    ResourceClient,
+    SourceRegistrar,
+    LocalSource,
+    SubscriptableTransformation,
+)
+from .serving import ServingClient
+
+
+class Client(ResourceClient, ServingClient):
+    """
+    Client for interacting with Featureform APIs (resources and serving)
+
+    **Using the Client:**
+    ```py title="definitions.py"
+    import featureform as ff
+    from featureform import Client
+
+    client = Client("http://localhost:8080")
+
+    # Example 1: Get a registered provider
+    redis = client.get_provider("redis-quickstart")
+
+    # Example 2: Compute a dataframe from a registered source
+    transactions_df = client.compute_df("transactions", "quickstart")
+    """
+
+    def __init__(
+        self, host=None, local=False, insecure=False, cert_path=None, dry_run=False
+    ):
+        ResourceClient.__init__(
```

Comment:
this should be:
super(ResourceClient, self).__init__

Reply:
@simba-git, I'd originally written it this way; however, I ran into errors relating to differing params (e.g. `dry_run`) and wasn't able to access `impl` on instances of `Client`. After reading [this SO answer](https://stackoverflow.com/questions/26927571/multiple-inheritance-in-python3-with-different-signatures) it seemed to me that I was reaching for the wrong tool for this job. Let me know what you think.

Reply:
Yeahhh I think since this is only temporary it's fine. The more idiomatic way is to use super but it's chaotic just like the SO answer suggested